apps/ramp-network/src/ramp.ts

```diff
@@ -13,6 +13,11 @@ export const ASSETS_BY_CHAIN: { [key: string]: string } = {
   '137': 'MATIC_*',
```

Comment:
Could you also check if it's possible to enable ERC20 here?

Reply:
How can I test that it is working?

Reply:
Try to buy USDC on Polygon

Reply:
do you have a polygon address I can use to impersonate?